mlflow/tracking/utils.py

```diff
@@ -164,7 +141,7 @@ def _is_databricks_uri(uri):
     return scheme == 'databricks' or uri == 'databricks'
 
 
-def _get_file_store(store_uri):
+def _get_file_store(store_uri, artifact_uri):  # pylint: disable=W0613
```

Comment:
Although these methods are internal to mlflow, and can thus be coversioned with changes to the registry, it may be a good idea to treat them as if they were in an external plugin, for consistency.

I would imagine that if I were writing an external plugin, I might write this something like

```
def _get_file_store(store_uri, **_):
```

This would allow this plugin to work with future versions of mlflow which may pass new arguments (e.g., if we added `store_token` today, it would break historical plugins that only take store_uri and artifact_uri). It also avoids the pylint warning (although `def _get_file_store(store_uri, **kwargs)` is more readable/expected, so may be reasonable to keep that).

Thoughts?

Reply:
I've changed it to `**_`.